LoveMachine.Core/ButtplugController.cs

```diff
@@ -195,6 +225,23 @@ protected void DoVibrate(float intensity, int girlIndex, Bone bone)
             client.VibrateCmd(intensity, girlIndex, bone);
         }
 
+        protected internal IEnumerator DoRotate(int girlIndex, Bone bone, bool clockwise, float strokeTimeSecs)
+        {
+            if (strokeTimeSecs == 0f)

```

Comment:
I think it's bad practice to treat 0 as a special value. Instead, please either call client.RotateCmd directly when the rotator needs to be stopped, or make a StopRotator method.